src/main/scala/viper/gobra/frontend/Parser.scala

```diff
@@ -490,6 +492,16 @@ object Parser {
     lazy val block: Parser[PBlock] =
       "{" ~> repsep(statement, eos) <~ eos.? <~ "}" ^^ PBlock
 
+    lazy val blockWithoutBraces: Parser[PBlock] =
+      repsep(statement, eos) <~ eos.? ^^ PBlock
+
+    lazy val blockWithBodyParameterInfo: Parser[(PBodyParameterInfo, PBlock)] =
+      "{" ~> bodyParameterInfo ~ blockWithoutBraces <~ "}"
+
+    lazy val bodyParameterInfo: Parser[PBodyParameterInfo] =
+      Constants.SHARE_PARAMETER_KEYWORD ~> repsep(idnUse, ",") <~ eos.? ^^ PBodyParameterInfo |
```

Comment:
Share**d**?

Reply:
What do you mean?

Reply:
I thought the keyword is `shared` and hence the constant should be named `SHARED_PARAMETER_KEYWORD`